src/components/CreateProposal/CreateProposalContainer.tsx

```diff
@@ -108,6 +109,11 @@ class CreateProposalContainer extends React.Component<IProps, null> {
 
   public render() {
     const { dao } = this.props;
+    const proposalDescriptions = dao.proposals
+      .map((proposal) => proposal as IProposalState)
```

Comment:
theres no more efficient way to do this?

Reply:
like?

Reply:
dao.proposals as Array<IProposalState> or something?

Reply:
corrected: https://github.com/daostack/alchemy/pull/102/commits/bef872213accb563d8b8a51c8258e9930867ee5a